lib/fog/compute/google/models/images.rb

```diff
@@ -68,6 +68,25 @@ def get(identity)
           new(data)
         end
 
+        def get_from_family(family)
+          # Search own project before global projects
+          all_projects = [service.project] + global_projects
```

Comment:
@icco Yeah, I took a look at that earlier, but it seems to match.

Ours:
```
 GLOBAL_PROJECTS = [
          "centos-cloud",
          "coreos-cloud",
          "debian-cloud",
          "google-containers",
          "opensuse-cloud",
          "rhel-cloud",
          "suse-cloud",
          "ubuntu-os-cloud",
          "windows-cloud"
        ]
```
Docs:
```
centos-cloud
coreos-cloud
debian-cloud
opensuse-cloud
rhel-cloud
suse-cloud
ubuntu-os-cloud
windows-cloud
```
Or am I missing something?

On method - good call, thanks! Just to be clear - you mean adding a private method, something like this?

```
        private

        def all_projects
          global_projects + [@project]
        end
```